effekt/jvm/src/test/scala/effekt/LLVMTests.scala

```diff
@@ -33,36 +31,17 @@ class LLVMTests extends EffektTests {
    * Documentation of currently failing tests and their reason
    */
   lazy val missingFeatures: List[File] = List(
-
-    // now show instance for records / datatypes
-    examplesDir / "pos" / "builtins.effekt",
-    examplesDir / "pos" / "namespaces.effekt",
-    examplesDir / "pos" / "triples.effekt",
-    examplesDir / "pos" / "either.effekt",
-
     // inspect
-    examplesDir / "pos" / "probabilistic.effekt",
-    examplesDir / "pos" / "nim.effekt",
     examplesDir / "pos" / "exists.effekt",
```

Comment:
I have some difficulties writing an elegant show function for `exists.effekt`, does anyone have a good idea?

Reply:
You can just change the type to store it:

```
type Exists {
  Op[Z](el: Z, combine: (Z, Z) => Z at {}, show: Z => String)
}
```